Add installation script

hwittenborn opened the pull request:
This is to help newer users who might not be able to get around the documentation just have a single command that sets up makedeb and (optionally) the Prebuilt-MPR on their system.

We'll set up a URL on our shlink instance so all the user has to run is this:

```sh
bash -c "$(wget -qO 'https://shlink.makedeb.org/install')"
```

hwittenborn committed:
Add install script

Elsie19 commented on install.sh:
```diff
@@ -0,0 +1,121 @@
+set -e
+
+# Handy env vars.
+hw_url='hunterwittenborn.com'
```
```suggestion
readonly hw_url='hunterwittenborn.com'
```

Elsie19 commented on install.sh:
```diff
@@ -0,0 +1,121 @@
+
+# Answer vars.
+SETUP_PREBUILT_MPR=0
+MAKEDEB_PKG='makedeb'
```
```suggestion
readonly MAKEDEB_PKG='makedeb'
```
Since it's always the same package

Elsie19 commented on install.sh:
```diff
@@ -0,0 +1,121 @@
+
+# Handy functions.
+msg() {
+    echo "${color_blue}[>]${color_normal} ${1}"
```
```suggestion
    echo "${color_blue}[>]${color_normal} ${@}"
```

Elsie19 commented on install.sh:
```diff
@@ -0,0 +1,121 @@
+}
+
+error() {
+    echo "${color_red}[!]${color_normal} ${1}"
```
```suggestion
    echo "${color_red}[!]${color_normal} ${@}"
```

Elsie19 commented on install.sh:
```diff
@@ -0,0 +1,121 @@
+}
+
+question() {
+    echo "${color_purple}[?]${color_normal} ${1}"
```
```suggestion
    echo "${color_purple}[?]${color_normal} ${@}"
```

Elsie19 commented on install.sh:
```diff
@@ -0,0 +1,121 @@
+}
+
+die_cmd() {
+    error "${1}"
```
```suggestion
    error "${@}"
```

hwittenborn committed:
Switch to using 'wget'

Both Debian and Ubuntu have wget installed out of the box, so we'll just use that.

hwittenborn committed:
Add quick links to end of installation instructions
